src/components/ReadPage.tsx

```diff
@@ -144,13 +144,22 @@ const ReadPage = ({ id }: Props) => {
                 userId={userId}
             />
             <div className={`${styles.controlBar}`}  >
-                <button
-                    onClick={() => router.push(`/comic?id=${parentPanelSet?.id}`)}
-                    style={backVisibility}
-                    id={`${styles.backButton}`}
-                >
-                    <img src={backIcon} className={`${styles.buttonIcon}`} alt="back button" />
-                </button>
+                <OverlayTrigger trigger={["focus", "hover"]} placement="bottom" overlay={parentPanelSet ? (<div></div>):(
+                    <Popover>
+                        <Popover.Body>
+                            You are at the start of the comic and cannot go back any further.
+                        </Popover.Body>
+                    </Popover>
+                )}>
+                    <button
+                        disabled = {!parentPanelSet}
+                        onClick={() => router.push(`/comic?id=${parentPanelSet?.id}`)}
+                        style={backVisibility}
+                        id={`${styles.backButton}`}
+                    >
+                        <img src={backIcon} className={`${styles.buttonIcon}`} alt="back button" />
+                    </button>
+                </OverlayTrigger>
```

Comment:
Not working on my end. No reminder shows up when I click on the trunk back button or when I hover over it. Maybe the button being disabled stops the triggers from happening?